examples/quotes/backend/quotes.py

```diff
@@ -0,0 +1,178 @@
+import asyncio
+import csv
+import os
+from time import time
+from typing import Annotated
+
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel, Field, ValidationError
+from sentence_transformers import SentenceTransformer
+
+from elasticsearch import NotFoundError
+from elasticsearch.dsl.pydantic import AsyncBaseESModel
+from elasticsearch import dsl
+
+model = SentenceTransformer("all-MiniLM-L6-v2")
+dsl.async_connections.create_connection(hosts=[os.environ['ELASTICSEARCH_URL']])
+
+
+class Quote(AsyncBaseESModel):
+    quote: str
+    author: Annotated[str, dsl.Keyword()]
+    tags: Annotated[list[str], dsl.Keyword()]
+    embedding: Annotated[list[float], dsl.DenseVector()] = Field(init=False, default=[])
+
+    class Index:
+        name = 'quotes'
+
+
+class Tag(BaseModel):
+    tag: str
+    count: int
+
+
+class SearchRequest(BaseModel):
+    query: str
+    filters: list[str]
+    knn: bool
+    start: int
+
+
+class SearchResponse(BaseModel):
+    quotes: list[Quote]
+    tags: list[Tag]
+    start: int
+    total: int
+
+
+app = FastAPI(
+    title="Quotes API",
+    version="1.0.0",
+)
+
+
+@app.get("/api/quotes/{id}")
+async def get_quote(id: str) -> Quote:
+    doc = None
+    try:
+        doc = await Quote._doc.get(id)
+    except NotFoundError:
+        pass
+    if not doc:
+        raise HTTPException(status_code=404, detail="Item not found")
+    return Quote.from_doc(doc)
```

Comment:
Isn't this code the same?

```suggestion
    try:
        doc = await Quote._doc.get(id)
        return Quote.from_doc(doc)
    except NotFoundError:
        raise HTTPException(status_code=404, detail="Item not found")
```

This also applies to `get_quote` and `delete_quote`.